hooks/useContractorInterest.ts

```
@@ -0,0 +1,189 @@
+  const createContractor = useCallback(async (data: any) => {
+    try {
+      await InterestedContractorsService.createContractor(data, accessToken);
+      refreshContractors();
+      return { success: true };
+    } catch (err) {
+      const message = err instanceof Error ? err.message : "Failed to create contractor";
+      return { success: false, error: message };
+    }
+  }, [refreshContractors, accessToken]);
+
+  const updateContractor = useCallback(async (id: string, data: any) => {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)